templates/dashboard.html

```diff
@@ -10,6 +10,8 @@
                 <h5 class="card-title">{{ config["ASSET_NAME"] }} balance</h5>
                 <p class="card-text">
                 {{ data["asset_balance"] }}
+		<center><a href="{{ data["wavesexplorer"] }}/address/{{ data["asset_address"]}}/tx">{{ data["premio_qrcode"] | safe }}</a><br/></center>
```

Comment:
it looks like you are mixing tabs and spaces here, also just link to the address on the waves explorer (not the txs)

Reply:
im not sure i understand, you mean having the the url for the waves explorer?

Reply:
no i mean remove the 'tx' at the end of the url

Reply:
>address *on* the waves explorer

not address *of* the waves explorer